user_logic.py

```diff
@@ -0,0 +1,36 @@
+import db
```

Comment:
user_logic.py module should be renamed to users_logic.py, since that is already in use and should not create an additional module in vain.